tide_search.install

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Install file.
+ */
+
+use Drupal\search_api\Item\Field;
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function tide_search_update_dependencies() {
```

Comment:
Is this actually required? Doesn't hurt to have it I guess.

Reply:
Without tide_core_8007, `field_topic:entity:path` and 'field_tags:entity:path' are always empty. Nothing to index.